src/Application/Application.csproj

```diff
@@ -6,4 +6,8 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Folder Include="Studen\" />
```

Comment:
you are missing a t in Student\

Reply:
I even deleted it. Wanted to have a new branch for it